etna/transforms/math/__init__.py

```diff
@@ -0,0 +1,17 @@
+from etna.transforms.math.add_constant import AddConstTransform
+from etna.transforms.math.lags import LagTransform
+from etna.transforms.math.log import LogTransform
+from etna.transforms.math.power import BoxCoxTransform
+from etna.transforms.math.power import YeoJohnsonTransform
+from etna.transforms.math.scalers import MaxAbsScalerTransform
+from etna.transforms.math.scalers import MinMaxScalerTransform
+from etna.transforms.math.scalers import RobustScalerTransform
+from etna.transforms.math.scalers import StandardScalerTransform
+from etna.transforms.math.sklearn import SklearnTransform
```

Comment:
If it does not intended for users it should have underscore before its name

Reply:
the same